com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using System.Collections.Generic;
+using Unity.Barracuda;
+using Unity.MLAgents.Actuators;
+using Unity.MLAgents.Inference;
+using Unity.MLAgents.Policies;
+using Unity.MLAgents.Sensors;
+using UnityEditor;
+using UnityEditor.Analytics;
```

Comment:
this will probably break since it's in the `Runtime` folder.

Reply:
I need to do another pass on this. What I'd like to do is keep the structure of the code editor-agnostic, and just wrap the EditorAnalytics.RegisterEventWithLimit / EditorAnalytics.SendEventWithLimit and have them be no-ops otherwise.